### ISSUE TITLE
Blobs can place over a dozen strong blobs on one tile

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#28811]
(WEB REPORT BY: kromkar REMOTE: 206.221.180.138:7777)
# Revision
11265d4fb52e0da1bd28638e478124aaafa405cb

# Description
What it says on the tin
# Steps to Reproduce
Be blob, place infinite walls
# What you Expected
To be able to defeat blob
# What Actually Happened
Unkillable infiniblob

**Comment by gurfan** [comment]
this is just because the right clicking is showing the blobs from adjacent tiles

**Comment by Dacendeth** [comment]
Yeah it's just a right click bug, if you alt click a tile you can see it's all normal.

**Comment by Blithering** [comment]
The big blob sprites are 64x64, which causes them to bleed over to surrounding tiles. Blob is ridiculously strong in many ways but this isn't one.

**Comment by DeityLink** [comment]
I wouldn't discard this report so easily, multiple blobs on a single tile have been an issue in the past. The right click thing is only valid if there's up to 9 blob showing up. If there's more than 9 then it's not just their sprite bleeding over.

TL;DR need more info, preferably even screenshots.

**Comment by DeityLink** [comment]
Not sure what I expected, anyway just open a new issue if it every actually occurs.
